src/Service/Signalement/SignalementFileProcessor.php

```diff
@@ -49,6 +51,12 @@ public function process(array $files, string $inputName): array
                             $this->filenameGenerator->generate($file)
                         );
                         $title = $this->filenameGenerator->getTitle();
+
+                        if (\in_array($file->getMimeType(), ImageManipulationService::IMAGE_MIME_TYPES)) {
+                            $this->imageManipulationService->setTmp(false);
+                            $this->imageManipulationService->resize($filename);
+                            $this->imageManipulationService->thumbnail($filename);
```

Comment:
Le resize et la génération d'image se fait à partir de la même image j'ai l'impression, pour éviter le coté répétitif, 
j’initialiserai l'image dans une nouvelle méthode, ça évitera d'ajouter le nom de l'image pour chaque traitement

Donc je verrai plutôt ça en terme d'usage mais rendant les paramètres optionnel afin de continuer à les utiliser ainsi si c'est nécéssaire

`$nomService->setImage('path/to/image.jpg')->resize()->thumbnail();`

Reply:
ok